the-blue-alliance-ios/Core Data/Match/Match.swift

```diff
@@ -169,6 +169,15 @@ extension Match {
     var blueAllianceTeamNumbers: [String] {
         return blueAllianceTeamKeys.map({ Team.trimFRCPrefix($0) })
     }
+    
+    /**
+     Returns the team keys from both alliances
```

Comment:
nit: `Returns the team keys from both alliances` -> `Returns the team keys for all alliances` once we implemented feedback of using `alliances`